Formula/b/bde.rb

```diff
@@ -26,14 +26,14 @@ class Bde < Formula
   depends_on "pcre2"
 
   resource "bde-tools" do
-    url "https://github.com/bloomberg/bde-tools/archive/refs/tags/4.8.0.0.tar.gz"
-    sha256 "49fdfb3a3e2c4803ba8a9bfa680cb50943c41ef1e6b1725087b877557b82bd35"
+    url "https://github.com/bloomberg/bde-tools/archive/refs/tags/4.13.0.0.tar.gz"
+    sha256 "d70ab85eb1a4325f3d569a6b7ea0f0a44a6143fd91905ab5fbaa5e1fed111a68"
   end
 
   def install
-    # TODO: `bde-tools` did not have a matching tag for 4.14.0.0. Check if it's in sync again in the next release.
+    # TODO: `bde-tools` did not have a matching tag since 4.14.0.0. Check if it's in sync again in the next release.
     # odie "bde-tools resource needs to be updated" if version != resource("bde-tools").version
-    odie "Check if bde-tools resource version is in sync again" if version > "4.14.0.0"
+    odie "Check if bde-tools resource version is in sync again" if version > "4.16.1.0"
```

Comment:
I'm on the fence if we should keep updating this line or accept that the tags don't match anymore and remove it.

Reply:
I think we should remove it, see the discussions in here, https://github.com/bloomberg/bde-tools/issues/52